### PR TITLE
Updating for v9: update CI to Github Actions

### PR DESCRIPTION
**Pull request by Ranieri93**
Hello!
Here I am with the PR relative to the CI update
@jsumners

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [18]
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
```
Comment (jsumners) [inline .github/workflows/ci.yaml:23]:
```suggestion
        uses: actions/checkout@v4
```

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v2
```
Comment (jsumners) [inline .github/workflows/ci.yaml:26]:
```suggestion
        uses: actions/setup-node@v3
```

File: .github/workflows/ci.yaml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    paths-ignore:
+      - '*.md'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [18]
```
Comment (jsumners) [inline .github/workflows/ci.yaml:19]:
```suggestion
        node-version: [18, 20]
```

**Review by jsumners — COMMENTED** [review]
Looking good. A few suggestions to keep us up-to-date.

**Comment by Ranieri93** [comment]
@jsumners Thank you so much for the suggestions!
If everything is settled here, what are the next steps? Should I open a new PR with the dependencies changes?

**Review by jsumners — APPROVED** [review]
Looks good to me.

**Comment by jsumners** [comment]
Yep. That would be a good next PR. We want to keep PRs focused so that they are easier to review.

**Pull request merged**